packages/cli-repl/src/run.ts

```diff
@@ -12,6 +12,7 @@ function enableFipsIfRequested() {
   }
 }
 enableFipsIfRequested();
+(process as any).noDeprecation = true;
```

Comment:
I don't think we need this anymore, since we replaced `new Buffer()` in `socksv5`